variation-commons-mongodb/src/main/java/uk/ac/ebi/eva/commons/mongodb/filter/VariantRepositoryEndFilter.java

```diff
@@ -0,0 +1,12 @@
+package uk.ac.ebi.eva.commons.mongodb.filter;
```

Comment:
Please add a copyright / licensing notice to new files, using other files in the repository as reference. The starting year should be 2019, and the copyright should belong to EBI.

Reply:
I have added the copyright / licensing notice to the new files.
Thank you :+1: